docs/collectors/EtcdCollector.md

```diff
@@ -0,0 +1,37 @@
+<!--This file was generated from the python source
+Please edit the source to make changes
+-->
+EtcdCollector
+=====
+
+
+Collects metrics from an Etcd instance.
+
+#### Example Configuration
+
+```
+    host = localhost
+    port = 2379
+```
+
+#### Options
+
+Setting | Default | Description | Type
+--------|---------|-------------|-----
+byte_unit | byte | Default numeric output(s) | str
+ca_file |  | Only applies when use_tls=true. Path to CA certificate file to use for server identity verification | str
+enabled | False | Enable collecting these metrics | bool
+host | localhost | Hostname | str
+measure_collector_time | False | Collect the collector run time in ms | bool
+metrics_blacklist | None | Regex to match metrics to block. Mutually exclusive with metrics_whitelist | NoneType
+metrics_whitelist | None | Regex to match metrics to transmit. Mutually exclusive with metrics_blacklist | NoneType
+port | 2379 | Port (default is 2379) | int
+timeout | 5 | Timeout per HTTP(s) call | int
+use_tls | False | Use TLS/SSL or just unsecure (default is unsecure) | bool
+
+#### Example Output
+
+```
+__EXAMPLESHERE__
```

Comment:
can you run `test.py -c etcdstat` since you have `setDocExample` in the test? This fills in this section